src/openrct2-ui/windows/Window.h

```diff
@@ -190,3 +190,11 @@ void window_tooltip_open(rct_window* widgetWindow, rct_widgetindex widgetIndex,
 void window_tooltip_close();
 
 rct_window* window_scenery_scatter_open();
+
+// clang-format off
+// close button on right of window
+#define WINDOW_SKELETON(TITLE, WIDTH, HEIGHT) \
+      WWT_FRAME,    0,  0,          WIDTH - 1, 0, HEIGHT, 0xFFFFFFFF,  STR_NONE }, \
+    { WWT_CAPTION,  0,  1,          WIDTH - 2, 1, 14,     TITLE,       STR_WINDOW_TITLE_TIP }, \
+    { WWT_CLOSEBOX, 0,  WIDTH - 13, WIDTH - 3, 2, 13,     STR_CLOSE_X, STR_CLOSE_WINDOW_TIP
```

Comment:
Wouldn't it be clearer to encapsulate this all with `{}`? That way it looks less like it's a define for just one widget, and is more in line with how `SPINNER_WIDGETS` works.

Reply:
Hm, I changed it because I thought this was more in line with how the rest of the code did it :').

I have to admit I did overlook SPINNER_WIDGET. Anyway, I will change it to encapsulate it with `{}`.